database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/PostDetailView.swift

```diff
@@ -17,7 +17,10 @@
 import SwiftUI
 
 struct PostDetailView: View {
-  var post: PostViewModel
+  @StateObject var post: PostViewModel
+  @State private var comment: String = ""
```

Comment:
@ryanwilson @morganchen12  @paulb777 Do you have an opinion if we should maintain this state in the view or it should be managed by the ViewModel. Treating it as a View state seems fine to me but I want to double with SwiftUI experts.

Reply:
This should be in the view since it's related to a control in the view itself - not data in the post (yet). The lifetime of the variable only lives as long as the view is on screen, where the PostViewModel definitely lives outside of it.

Reply:
Yeah, good point 👍 . I could imagine the comment draft being saved to DB as it's being typed, then we would need it to be a part of ViewModel, but with the current implementation local private state makes better sense.

Reply:
Yep, I think this is fine. It's view state until it gets written to the database.